oak_functions/client/java/com/google/oak/functions/client/AttestationClient.java

```diff
@@ -38,13 +40,16 @@ public AttestationClient(String uri) throws Exception {
         RemoteAttestationStub stub = RemoteAttestationGrpc.newStub(channel);
 
         // Create server response handler.
-        messageQueue = new ArrayBlockingQueue<AttestedInvokeResponse>(1);
+        messageQueue = new ArrayBlockingQueue<>(1);
         StreamObserver<AttestedInvokeResponse> responseObserver = new StreamObserver<AttestedInvokeResponse>() {
```

Comment:
```suggestion
        StreamObserver<AttestedInvokeResponse> responseObserver = new StreamObserver<>() {
```
Could `AttestedInvokeResponse` be removed here too?

Reply:
I think no.
The type should be specified at least once. Otherwise it would become a raw Java type.

Reply:
I think since the type is specified in the declaration of the variable, it is not necessary in the instantiation statement.

Reply:
Done

Reply:
Ah! I did not realize this is an anonymous type implementation. Both types are needed. I have created a PR to fix this: #2159